features/fixtures/simple_app/bin/crystalball

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'crystalball'
+require_relative '../standard_predictor'
+
+Crystalball::RSpec::Runner.invoke
```

Comment:
I guess you don't need this file at all. You can specify proper predictor class in config